modules/custom/az_global_footer/az_global_footer.install

```diff
@@ -139,3 +139,36 @@ function az_global_footer_update_1020703() {
 
   return t('Updated %count menu link(s) from A-Z Index to Directory in the global footer.', ['%count' => $updated_count]);
 }
+
+/**
+ * Update Twitter link(s) to new X link(s)
+ *
+ * Twitter icon(s) in the global footer will be updated with
+ * new X icon(s).
+ */
+function az_global_footer_update_1020704() {
```

Comment:
Update the function name; needs to reflect patch release

Reply:
https://github.com/az-digital/az_quickstart/blob/main/RELEASES.md#update-hook-numbering-convention

Reply:
looks like it needs to be 1020801